frontend/package.json

```diff
@@ -62,5 +62,9 @@
     "webpack": "^4.40.2",
     "webpack-cli": "^3.3.0",
     "webpack-dev-server": "^3.8.1"
+  },
+  "dependencies": {
+    "@types/react-table": "^6.8.5",
+    "react-table": "^6.10.3"
```

Comment:
`react-table` is not used except for the CSS file. Please don't add dependencies when they're not needed yet. Also, they should be in `devDependencies`, since we only have a build step for the frontend, no runtime.